bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/internal/clangd/ClangdFallbackManager.java

```diff
@@ -46,31 +53,40 @@ class FallbackFlags {
 	}
 
 	private static final String ISYSTEM = "-isystem"; //$NON-NLS-1$
-	private final ServiceCaller<ICBuildConfigurationManager> configManagerServiceCaller = new ServiceCaller<>(
-			getClass(), ICBuildConfigurationManager.class);
-	private final InitialFileManager initialFileManager = InitialFileManager.getInstance();
 	private final boolean isWindows = Constants.OS_WIN32.equals(Platform.getOS());
 
-	public FallbackFlags getFallbackFlagsFromInitialUri() {
+	@Reference
+	private ICBuildConfigurationManager configurations;
```

Comment:
Maybe call the field `cBuildConfigurationManager` since it is only a single value and it makes the code better readable to me.

Reply:
I would prefer to use single word variable, compound name indicates that the scope of variable is too big and complex.
Do you think `configuration` is better readable here? If not, how would you name it in single word manner?

Reply:
I would be fine with `configuration`, but `configManager`/ `configurationManager` would indicate that it's not a configuration, only the manager for it.

Reply:
> but configManager/ configurationManager would indicate that it's not a configuration, only the manager for it

This is what I tried to express with , "the one who manages configurations for you", i.e. with single word `configurations` in the very first variant. How would you name it using single noun?

Reply:
ok, I can offer name `build` for this field. Will it work for you @ghentschke ?